index.html

```diff
@@ -132,10 +133,10 @@ <h2>Sponsors</h2>
       </section>
 
 
-      <section>
+      <section id="footer">
```

Comment:
great job using unique identifies that explain what is taking place in the code.